src/main/flight/imu.c

```diff
@@ -31,6 +31,8 @@
 
 #include "common/axis.h"
 
+#include "config/config.h"
```

Comment:
I don't think this is needed.

Reply:
This include is required for `currentPidProfile->crash_recovery`